### PR TITLE
Feat/ota 2619/user switch organizations

### PR DESCRIPTION
**Pull request by piotrcichonmee**
[OTA-2619] Dropdown with organizations the user can switch to

### CODE REVIEW

File: ota-plus-web/app/reactapp/style/ota-default/override-ant.scss [hunk 1]
```diff
@@ -50,6 +50,29 @@ h3 {
     color: $color-white;
   }
 
+  &-hero.organizationNamespace {
```
Comment (piotrsmol) [inline ota-plus-web/app/reactapp/style/ota-default/override-ant.scss]: Please use standard `css` naming convention (words seperated by dash `-`)

File: ota-plus-web/app/reactapp/src/stores/UserStore.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 /** @format */
 
-import { observable } from 'mobx';
+import Cookies from 'js-cookie';
+import { observable, toJS } from 'mobx';
```
Comment (piotrsmol) [inline ota-plus-web/app/reactapp/src/stores/UserStore.js]: Remove unnecessary import `toJS`

Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/src/stores/UserStore.js]: Done

File: ota-plus-web/app/reactapp/style/ota-default/override-ant.scss (hunk 1 above)
Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/style/ota-default/override-ant.scss]: Done

**Pull request merged**